api-report/runtime-definitions.api.md

```diff
@@ -296,6 +296,7 @@ export interface ISummarizerNodeConfig {
 // @public (undocumented)
 export interface ISummarizerNodeConfigWithGC extends ISummarizerNodeConfig {
     readonly gcDisabled?: boolean;
+    readonly maxUnreferencedTime?: number;
```

Comment:
This is a duration, right? Sorry for being nitpicky about this but can you name it as such and also include the unit of measurement? Depending on someone's train of thought, this can be a unix timestamp of an ultimatum for an object's lifetime.

Reply:
Oh yeah, that makes sense. I should have taken care of this based on your comments in my other PR :)

Reply:
Actually, I remember changing all the names to have `ms` suffix in this PR itself. Something must have gone wrong during mering or something. I will change it again.